### ISSUE TITLE
golang-github-hashicorp-raft doesn't requires anything

### ISSUE THREAD

**Issue by matwey** [marguerite/golang-packaging#10]
Hello,

the following package doesn't require any others:

```
> rpm -q -R -p golang-github-hashicorp-raft-0.0.0+git20151117.d136cd1-1.3.x86_64.rpm
предупреждение: golang-github-hashicorp-raft-0.0.0+git20151117.d136cd1-1.3.x86_64.rpm: Заголовок V3 DSA/SHA1 Signature, key ID 307d7bf9: NOKEY
rpmlib(CompressedFileNames) <= 3.0.4-1
rpmlib(PayloadFilesHavePrefix) <= 4.0-1
rpmlib(PayloadIsLzma) <= 4.4.6-1
```

At the same time, the package provides file `/usr/lib64/go/contrib/pkg/linux_amd64/github.com/hashicorp/raft.a` which requires at least two external packages:

```
        import codec "github.com/hashicorp/go-msgpack/codec"
        import metrics "github.com/armon/go-metrics"
```

**Comment by marguerite** [comment]
err...

```
 [  119s] error: Couldn't exec %{_perfix}/lib/rpm/golang.req: No such file or directory
```

looks like a typo: s/perfix/prefix

**Comment by matwey** [comment]
Where did it come from? I don't see neither prefix  not perfix in the spec file.

**Comment by marguerite** [comment]
@matwey not your fault. mine. it's from golang.attr which tells rpm where to find the Requires finding script.

**Comment by marguerite** [comment]
fixed in 14.5 release
